src/middleware/auth.ts

```diff
@@ -25,7 +27,7 @@ const authenticateTenant = (req: Request, res: Response, next: NextFunction): vo
 
     next()
```

Comment:
Why change here to `token == null`?
![image](https://github.com/CRANKBIT/CrankBit-BackEnd/assets/68600416/328965c3-aec1-49de-ac51-02b3fcf84fa9)
'Bearer ' looks like will pass this middleware, security issue

Reply:
I'm testing a token and suspect that the issue I encountered, where the token couldn't be retrieved, might be related to the "Bearer" keyword. I'll revert the changes afterward.